plugins/quickinspector/quickinspector.cpp

```diff
@@ -906,6 +954,19 @@ void QuickInspector::registerMetaTypes()
     MO_ADD_PROPERTY(QSGOpacityNode, qreal, opacity, setOpacity);
     MO_ADD_PROPERTY(QSGOpacityNode, qreal, combinedOpacity, setCombinedOpacity);
 
+    MO_ADD_METAOBJECT1(QSGRenderNode, QSGNode);
+#if QT_VERSION >= QT_VERSION_CHECK(5, 8, 0)
+    MO_ADD_PROPERTY_RO(QSGRenderNode, QSGRenderNode::StateFlags, changedStates);
+    MO_ADD_PROPERTY_RO(QSGRenderNode, QSGRenderNode::RenderingFlags, flags);
+    MO_ADD_PROPERTY_RO(QSGRenderNode, QRectF, rect);
+    MO_ADD_PROPERTY_RO(QSGRenderNode, qreal, inheritedOpacity);
+#else
```

Comment:
That class is private before and is available since 5.3 at least. Though agreed to only support 5.8 and up.